releasing.rst

```diff
@@ -29,124 +27,89 @@ we do a few things to prepare:
 4. Review all of the outputs (e.g., plots)
    generated from the test suite for abnormalities.
 
-5. Run all tests for projects that depend on your project.
-   For example, Nengo OCL depends on Nengo core
-   so we run those tests with
-
-   .. code-block:: bash
-
-      pytest --pyargs nengo --plots --simulator nengo_ocl.Simulator
-
-   If any tests fail, attempt to fix them in your project.
-   If they cannot be fixed and require upstream changes,
-   then file an issue with that project or contact its maintainer
-   to determine how to proceed.
-
-6. Build documentation and review it for abnormalities.
+5. Build documentation and review it for abnormalities.
 
-7. Commit all changes from the above steps before moving on to the next stage.
+6. Commit all changes from the above steps.
 
-Note that any fixes done as a result of
-Stage 1 should be done through the normal process of making
-a pull request and going through review.
-However, from Stage 2 onward, the work is done directly
-on the ``master`` branch.
-Proceed with caution!
-
-Stage 2: Releasing
-==================
-
-Once everything is prepared, we're ready to do the release.
-It should be okay to work in the same directory that you
-do development, but if you want to be extra safe,
-you can do a fresh clone of the repository
-into a separate directory.
-
-1. Change the version information for your project.
+7. Change the version information for your project.
    For most Python projects, it lives in ``project/version.py``.
    See that file for details.
 
-2. Set the release date in the changelog (usually ``CHANGES.rst``).
-
-3. Run ``collective.checkdocs`` to ensure proper formatting for PyPI.
-
-   .. code-block:: bash
-
-      python setup.py checkdocs
+8. Set the release date in the changelog, ``CHANGES.rst``.
 
-4. ``git add`` the changes above and make a release commit with
+9. ``git add`` the changes above and make a release commit with
 
    .. code-block:: bash
 
       git commit -m "Release version X.Y.Z"
```

Comment:
This is super minor, but in NengoDL I started doing "Release vX.Y.Z", just to line up with the tag and the next "Starting development of vX.Y.Z+1" commit.  Definitely doesn't matter, just occurred to me as I was reading this.

Reply:
Yeah I've been annoyed by the inconsistency before too, but erred on the side of consistency within the commit log. But yeah I think I'm in favor of switching so I'll change that.